library/emqx

```diff
@@ -44,8 +44,14 @@ GitCommit: 35e70c8e602687db5a447c9573bde8ab77335fdc
 Directory: 5.7
 File: Dockerfile
 
-Tags: 5.8.0, 5.8, 5, latest
+Tags: 5.8.0
 Architectures: amd64, arm64v8
 GitCommit: 69e9c223e2215fb5d071dec6cc6aa19c106331e7
 Directory: 5.8
 File: Dockerfile
```

Comment:
Any particular reason to keep this version listed here?  Removing tags here will remove them from the "Supported" section on the Hub readme (and will prevent us from spending cycles rebuilding them on the official build servers), but the tags will still be available to users who want them.  (See https://github.com/docker-library/official-images#library-definition-files for more detail on this.)